protos/protos.d.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 Google LLC
+// Copyright 2025 Google LLC
```

Comment:
```suggestion
// Copyright 2024 Google LLC
```
per ospo guidance the year should be the year the file was authored